### PR TITLE
Feat improve graph metric extractor

### PR DESCRIPTION
**Pull request by sebglon**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+############################
+FROM scratch
+# Copy our static executable.
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=builder /go/bin/graphql-exporter /go/bin/graphql-exporter
+COPY config_example.yaml /config.yaml
+EXPOSE 9353
+# Run the hello binary.
+ENTRYPOINT ["/go/bin/graphql-exporter"]
+CMD ["-config_path", "/config.yaml"]
```
Comment (ClementGautier) [inline Dockerfile]: You have multiple files with improper end of file. Take the time to configure your IDE so it adds a new line at the end of files: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

File: cmd/graphql-exporter/main.go [hunk 2]
```diff
@@ -14,16 +15,52 @@ func main() {
...
+
+	}
+}
+
+func getenv(key, fallback string) string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return fallback
+	}
+	return value
```
Comment (ClementGautier) [inline cmd/graphql-exporter/main.go:65]: Maybe use https://pkg.go.dev/github.com/cbrewster/slog-env instead

File: internal/prometheus/collector.go [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/vinted/graphql-exporter/internal/config"
+	"github.com/vinted/graphql-exporter/internal/graphql"
+)
+
+var latencyHistogramBuckets = []float64{.1, .25, .5, 1, 2.5, 5, 10, 15, 20, 30, 40, 50, 60, 90, 150, 210, 270, 330, 390, 450, 500, 600, 1200, 1800, 2700, 3600}
```
Comment (ClementGautier) [inline internal/prometheus/collector.go]: Do we really need that much buckets for latency? you set a 60s timeout limit so everything > 60 seems unlikely, can we use the [default value](https://github.com/prometheus/client_golang/blob/1ba60c7d585210b1df898b4077ed8cf45b4bf0d2/prometheus/histogram.go#L54-L59) instead?

**Review by ClementGautier — CHANGES_REQUESTED** [review]

File: cmd/graphql-exporter/main.go (hunk 2 above)
Comment (sebglon) [reply cmd/graphql-exporter/main.go:65]: I am not super fan of the env naming on the library and adding an external lib only for one env seems to much

**Review by ClementGautier — APPROVED** [review]

**Pull request merged**